tortoise/contrib/pylint/__init__.py

```diff
@@ -33,7 +34,15 @@ def transform_model(cls):
     Also keep track of relationships and make them in the related model class.
     '''
```

Comment:
I missed it last time, but I think we should use double quotes in docstrings. Not like this is clear in PEP8, but even in it's [PEP](https://www.python.org/dev/peps/pep-0257/) all examples are with double quotes